tracing/tracing.go

```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 
 	"go.opentelemetry.io/otel"
+	//nolint:staticcheck
 	"go.opentelemetry.io/otel/exporters/jaeger"
```

Comment:
FYI @aluzzardi the jaeger exporter is deprecated, jaeger now says to just use plain otel. I had to add this to get lint passing.

Is this code still worth maintaining? I don't remember if it's from older iterations on metrics/logging or if it's still in use.